### ISSUE TITLE
Pass in already checked options

### ISSUE THREAD

**Issue by hbellala** [IAmNithi/multiselect-dropdown-react#1]
Is there a way to pass in already checked options? I am storing the info so when I go back is there a way I can pass the options already selected?

**Comment by priscilacoutinho** [comment]
Yeah... I am going through the same thing... Trying to get an option already checked by default

**Comment by IAmNithi** [comment]
Hi @hbellala ,
I'm currently Working on it with the latest dependencies. I will Close it ASAP.
